defaults/main.yml

```diff
@@ -9,10 +9,17 @@ sap_hana_deployment_bundle_sar_file_name:
 sap_hana_deployment_sapcar_path:
 sap_hana_deployment_sapcar_file_name:
 
+# if the previous variables are undefined because you do not want to provide a bundle
+# you can define sap_hana_installdir variable, which points to the directory containing hdblcm
+# the role sap-hana-mediacheck can be used to define this variable
```

Comment:
What do you mean by `sap-hana-mediacheck can be used to define this variable`? Are we forcing users to use that role to generate that variable? Adding a dependency from another Role is not a good practice. You should be able to use a Role independently to others

Reply:
Hi Mak,
you can set the variable to wherever hdblcm is located. If you use sap-hana-mediacheck to mount or unpackit this role also sets this variable. Thats what I mean. It's an option, not arequirement. But to be on the safe side I added the variable in defaults now